x-pack/plugins/security/public/management/roles/roles_grid/roles_grid_page.tsx

```diff
@@ -254,6 +253,21 @@ export class RolesGridPage extends Component<Props, State> {
           );
         },
       },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.security.management.roles.descriptionColumnName', {
+          defaultMessage: 'Role Description',
+        }),
+        sortable: true,
+        truncateText: { lines: 2 },
+        render: (description: string, record: Role) => {
+          return (
+            <EuiText color="subdued" size="s" data-test-subj={`roleRowDescription-${record.name}`}>
+              {description}
+            </EuiText>
```

Comment:
Nit: add a tooltip to see full description when text is truncated?

<img width="910" alt="Screenshot 2024-05-14 at 7 15 23 PM" src="https://github.com/elastic/kibana/assets/103939324/3cae910a-4fb7-4b21-8a36-6c1d5d63848d">

Reply:
Added tooltip and changed default truncation to 3 lines (seems our built in roles have quite a long description).